### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by fbclh**
**Project requirements:**

- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder The index.html file must be set as a template using the HTML Webpack Plugin.
- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys: description [string], completed [bool], index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
- All your source files (index.html, index.js and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

**Review by eri8-9 — COMMENTED** [review]
Hi @fbclh, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

wrong way

## Invalid Code Review Request

- [ ] Pull Request should have a descriptive title telling you what changes are inside.
- [ ] Pull Request should have a short summary describing introduced changes in more detail.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+.fa-ellipsis-v {
+  /* display: flex; */
+
+  /* align-self: flex-end; */
+
+  /* color: red; */
+
+  /* float: right; */
+}
```
Comment (Mihndim2020) [inline src/style.css:101]: - [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+.fa-ellipsis-v::before {
+  /* color: red; */
+
+  /* display: flex; */
+
+  /* align-self: flex-end; */
+
+  /* flex: 1; */
```
Comment (Mihndim2020) [inline src/style.css:90]: - [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+.refresh {
+  margin-right: 1rem;
+}
+
+.todo__container ul li .kebab {
+  /* color: grey;
+  float: right;
+  cursor: move; */
```
Comment (Mihndim2020) [inline src/style.css:80]: - [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  margin-left: 1rem;
+}
+
+.todo__root {
+  display: flex;
+  flex-direction: column;
+
+  /* gap: 2rem; */
+
+  /* padding: 2rem; */
```
Comment (Mihndim2020) [inline src/style.css:55]: - [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+.todo {
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.125rem;
+  box-shadow: rgb(0 0 0 / 24%) 0 3px 8px;
+  background-color: #fff;
+
+  /* width: 50%; */
+
+  /* padding: 1rem; */
```
Comment (Mihndim2020) [inline src/style.css:20]: - [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @fbclh, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Descriptive Readme file,
:heavy_check_mark: Your project is running free of `Linter` errors.


There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @fbclh, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**